afb-binding/src/verbs-frontend.rs

```diff
@@ -374,20 +374,27 @@ fn authorize_response(
         _ => return afb_error!("ocpp-authorize-start", "invalid response type"),
     };
 
+    let ocpp_auth_state;
     match response.id_tag_info.status {
         v106::AuthorizationStatus::Accepted => {
             afb_log_msg!(Debug, rqt, "ocpp-authorize-done");
-            ctx.mgr.authorized(true)?;
+            ocpp_auth_state = true;
+        }
+        v106::AuthorizationStatus::Blocked => {
+            afb_log_msg!(Debug, rqt, "ocpp-authorize-blocked");
+            ocpp_auth_state = false;
         }
         _ => {
```

Comment:
I agree. Since it was returning afb_error for those cases (Expired, Invalid ...) before we created the branch, we didn't want to change it. I will see it with Romain.

Reply:
Modif is done.